fiftyone/server/routes/groups.py

```
@@ -0,0 +1,334 @@
+                        if len(serialized_group) > 1
+                        else field_path
+                    )
+                    urls[full_path] = url
+
+        response_data = {"group": serialized_group}
+        if urls:
+            response_data["urls"] = urls
+
+        # If filepath wasn't explicitly requested, remove it from the response
+        if fields and "filepath" not in fields:
+            for slice_name_key, sample_data in serialized_group.items():
```

Ruff (0.14.3)

Loop control variable `slice_name_key` not used within loop body

(B007)